src/elements/Container/Container.js

```diff
@@ -1,33 +1,68 @@
-import React, { Component, PropTypes } from 'react'
-import classNames from 'classnames'
-import META from '../../utils/Meta'
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+import META from '../../utils/Meta';
+import { getUnhandledProps, useKeyOnly } from '../../utils/propUtils'
 
 /**
- * A container that gives your content some side padding.
+ * A container limits content to a maximum width
  */
-export default class Container extends Component {
-  static propTypes = {
-    children: PropTypes.node,
-    className: PropTypes.string,
-  }
-
-  static _meta = {
-    library: META.library.semanticUI,
-    name: 'Container',
-    type: META.type.element,
-  }
-
-  render() {
-    const classes = classNames(
-      'sd-container',
-      'ui',
-      this.props.className,
-      'container',
-    )
-    return (
-      <div {...this.props} className={classes}>
-        {this.props.children}
-      </div>
-    )
-  }
+function Container(props) {
+  const {
+    text, left, center, right, justified, fluid,
+    children, className,
+  } = props
+
+  const classes = cx('sd-container ui',
+    useKeyOnly(text, 'text'),
+    useKeyOnly(left, 'left aligned'),
+    useKeyOnly(center, 'center aligned'),
+    useKeyOnly(right, 'right aligned'),
+    useKeyOnly(justified, 'justified'),
```

Comment:
There are 4 types of possible class to props mappings for SUI classes.  The alignment class here follows this API pattern:

```js
aligned="left"    // useValueAndKey(aligned, 'aligned')
aligned="right"   // useValueAndKey(aligned, 'aligned')
aligned="center"  // useValueAndKey(aligned, 'aligned')
```

One reason we use this key/value relationship is because two simultaneous alignment props are not valid:

```
<Container right left />
```

Where as making those values of the `aligned` prop means you can only have one at a time:

```
<Container aligned="left" />
```

However, the `justified` option breaks down however since the resulting class is not `justified aligned`.  We've not yet run into this matchup yet so I'm open to suggestions.  The best immediate thought I have is to still allow `aligned="justified"` for the API consistency, then in cx buildup:

```js
cx(
  useValueAndKey(aligned, 'aligned'),
  useValueAndKey(aligned, 'aligned'),
  useValueAndKey(aligned, 'aligned'),
  aligned === 'justified' && 'justified'
)
```

Thoughts?